docs/ops.md

```diff
@@ -19,19 +19,36 @@ USAGE:
    containerd [global options] command [command options] [arguments...]
 
 VERSION:
-   v1.0.0-alpha3-36-ge9b86af
+   v1.7.0
```

Comment:
We are still in v1.7.0-beta.0. I think we should align with it.

Reply:
ah,thanks for review.. updated again
@fuweid